pkg/where/where.go

```diff
@@ -0,0 +1,147 @@
+package where
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/lib/pq"
+)
+
+// By default all rows are returned.
+// This can be limited by providing one or more rows= query variables.
+//
+// Each rows variable looks like this:
+//
+//	rows=col:rowspec[,rowspec]
+//
+// Where col is a column name, and rowspec is one of:
```

Comment:
can rowspec be an array?

Reply:
As it stands, rowspec is either a single geographic id or two ids separated by "...". Multiple rowspecs are separated by commas.

Reply:
so would `rows=geography_code:e1,e2,e3` be a valid query? (if those were valid geography codes)